db/migrate/20240927173326_create_establishments.rb

```diff
@@ -0,0 +1,13 @@
+class CreateEstablishments < ActiveRecord::Migration[7.1]
+  def change
+    create_table :establishments do |t|
+      t.string :name, null: false
+      t.references :address, null: false, foreign_key: true
+      t.decimal :latitude, precision: 10, scale: 6, null: false
+      t.decimal :longitude, precision: 10, scale: 6, null: false
+      t.decimal :maps_location, precision: 10, scale: 6
```

Comment:
`maps_location` should be string. I am thinking it can be google maps URL.

Reply:
will do.